jupyter_ydoc/ydoc.py

```diff
@@ -10,6 +10,12 @@
 
 from .utils import cast_all
 
+# The minimum major version of the notebook format we support.
+NBFORMAT_MAJOR_VERSION = 4
+
+# The minimum minor version of the notebook format we support.
+NBFORMAT_MINOR_VERSION = 4
```

Comment:
It makes more sense to set these as default notebook value as this is what we are doing with it:

```suggestion
# The default major version of the notebook format.
NBFORMAT_MAJOR_VERSION = 4

# The default minor version of the notebook format.
NBFORMAT_MINOR_VERSION = 5
```